shared/cfg/guest-os/Windows/Win2012/x86_64.cfg

```diff
@@ -8,9 +8,10 @@
         sha1sum_cd1 = d3fd7bf85ee1d5bdd72de5b2c69a7b470733cd0a
         sha1sum_1m_cd1 = 9194a3aabae25b36e5f73cad001314b2c8d07d14
         unattended_file = unattended/win2012-autounattend.xml
+        ovmf:
+            unattended_file = unattended/win2012-autounattend_ovmf.xml
```

Comment:
the answer file name for win2012 and win2016 are not the in the same foramt:
unattended/win2012-autounattend_ovmf.xml
unattended/win2016-64-autounattend_ovmf.xml

Reply:
@lijinlijin  yes,  it is different format. 
Before, For upstream, it use unattended_file  "win2016-64-autounattend.xml" in cfg,  but add "win2016-autounattend_ovmf.xml",  this is a bug for upstream, refer commit 7b6e30677c80db2707ef32e5ac53a6655acd552b.
But, in staf, file name is "win2016-64-autounattend.xml", so we can run this case. 

I just want to keep consistent.  If you think "win2016-autounattend.xml"  is better, i will change the params in cfg instead of the file name. And staf will be updated accordingly. Thanks !

Reply:
After discussion with yfu, she decides to modify win2016 and win2019 name to keep consistent with win2012.
As there is no 32 bit platform for win2012, win2016, win2019, no need to add "64" string.

Reply:
@lijinlijin  updated according to your advice,  don't use "-64" for win2016 and win2019 which only have 64 bit platform. Please help review again, thanks a lot!  staf will be updated accordingly later.